pkg/operator/ceph/csi/spec.go

```diff
@@ -349,7 +357,7 @@ func createCSIDriverInfo(clientset kubernetes.Interface, name string) error {
 		return nil
 	}
 	if apierrors.IsAlreadyExists(err) {
-		logger.Info("CSIDriver CRD already had been registered for %q", name)
+		logger.Infof("CSIDriver CRD already had been registered for %q", name)
```

Comment:
This should be a separate commit since this has nothing to do with the original goal of the PR.

Reply:
done